apps_ci/scripts/bump_version.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import argparse
+import os
+import pathlib
+import yaml
+
+from apps_exceptions import AppDoesNotExist, ValidationErrors
+from apps_ci.version_bump import map_renovate_bump_type, bump_version, rename_versioned_dir
+
+
+def update_app_version(app_path: str, bump_type: str) -> None:
+    if not os.path.exists(app_path):
+        raise AppDoesNotExist(app_path)
+
+    verrors = ValidationErrors()
+    app_dir = pathlib.Path(app_path)
+    app_metadata_file = app_dir / 'app.yaml'
+    if not app_metadata_file.is_file():
+        verrors.add('app_metadata', 'app.yaml file is missing')
+
+    verrors.check()
+
+    with open(str(app_metadata_file), 'r') as f:
+        app_config = yaml.safe_load(f.read())
+
+    bump_type = map_renovate_bump_type(bump_type)
+
+    old_version = app_config['version']
+    app_config['version'] = bump_version(old_version, bump_type)
+    rename_versioned_dir(old_version, app_config['version'], app_dir.parent.name, app_dir)
+
+    with open(str(app_metadata_file), 'w') as f:
+        f.write(yaml.safe_dump(app_config))
+
+    print(
+        f'[\033[92mOK\x1B[0m]\tUpdated app {app_dir.name!r} version from {old_version!r} to {app_config["version"]!r}'
+    )
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--path', help='Specify path of the app to be updated')
+    parser.add_argument(
+        # We allow more choices here, as "Renovate" has more update types. We still map them to the ones we need.
+        '--bump', nargs=1, type=str, choices=('major', 'minor', 'patch', 'digest', 'pin', 'pinDigest'),
```

Comment:
Another neat property of this module is that you can actually do `type=map_renovate_bump_type` and it will call your function automatically and return whatever it returns.